### PR TITLE
Create soundrecorder.yml

### PR DESCRIPTION
**Pull request by naXa777**
propose new app

### CODE REVIEW

File: _data/projects/soundrecorder.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: SoundRecorder
+desc: A simple sound recording Android app implementing Material Design. The goal of this project is to develop as stable as possible sound recorder application.
+site: https://github.com/naXa777/SoundRecorder
```
Comment (ritwik12) [inline _data/projects/soundrecorder.yml:3]: @naXa777 I can see that this is a fork, are you sure that you want to add a fork? you can also move your fork to a new repo if the owner of the repo doesn't have any problem.

File: _data/projects/soundrecorder.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Gradle
+- Android
+- mobile
+- sound-record
+- audio-recorder
+- dictaphone
+- material-design
+upforgrabs:
+  name: help wanted
+  link: https://github.com/naXa777/SoundRecorder/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (ritwik12) [inline _data/projects/soundrecorder.yml]: @naXa777 Please change this to https://github.com/naXa777/SoundRecorder/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/soundrecorder.yml (hunk 1 above)
Comment (naXa777) [reply _data/projects/soundrecorder.yml:3]: And what's wrong with forks? It's a fork of https://github.com/dkim0419/SoundRecorder which is licensed under [GNU v3](https://github.com/dkim0419/SoundRecorder/blob/master/LICENSE) so modifications and distribution are permitted.

File: _data/projects/soundrecorder.yml (hunk 2 above)
Comment (naXa777) [reply _data/projects/soundrecorder.yml]: Thank you! Changed.

File: _data/projects/soundrecorder.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/soundrecorder.yml:3]: @naXa777 There is no problem as long as you feel fine with it.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@naXa777 Thanks!!

**Pull request merged**